docs/src/user-guide/command-line-interface.md

```diff
@@ -154,9 +103,14 @@ npx eslint --env browser --env node file.js
 #### `--ext`
 
 This option allows you to specify which file extensions ESLint uses when searching for target files in the directories you specify.
-By default, ESLint lints `*.js` files and the files that match the `overrides` entries of your configuration.
 
-Examples:
+* **Argument Type**: String. File glob pattern.
```

Comment:
Expected format is `.<extension>` (for example, `.ts`), I think this shouldn't be described as "File glob pattern".

Reply:
fixed in 89113ac71e2f71d509c499b3d5883e643228e149